get_dependencies.py

```diff
@@ -24,11 +24,22 @@ def download(url, filename):
     sys.stdout.write('\n')
 
 
-print('[*] Extracting webdriver archives...')
-for f in ['geckodriver', 'nightly', 'chromedriver', 'chrome-linux']:
-    tar = tarfile.open('tools/%s.tar.xz' % f, 'r:xz')
-    tar.extractall(path='tools/')
-    tar.close()
+if sys.platform.startswith('linux'):
+    url = 'https://www.dropbox.com/s/cbg8mpg03vd8cph/linux.tar.xz?dl=1'
```

Comment:
Use `https://www.dropbox.com/s/ziti4nkdzhgwg1n/linux.tar.xz?dl=1` here (I've uploaded it myself)

Reply:
@marco-c  did you checked the files that i have uploaded?

Reply:
Yes, I've moved your files to my dropbox.